### PR TITLE
Modified Proto files

### PR DESCRIPTION
**Pull request by dheerajkd30**

### CODE REVIEW

File: proto/comdex/cdp/v1alpha1/msg.proto [hunk 1]
```diff
@@ -8,19 +8,19 @@ import "gogoproto/gogo.proto";
 option go_package = "github.com/comdex-official/comdex/x/cdp/types";
 
 // Msg defines the Msg service.
-service Msg {
+service MsgService {
```
Comment (prashantkrv2020) [inline proto/comdex/cdp/v1alpha1/msg.proto:11]: We don't need to write Service here. When the go generated code will add Server on top of it. Don't need to make it MsgServiceServer

File: proto/comdex/cdp/v1alpha1/query.proto [hunk 2]
```diff
@@ -12,36 +12,36 @@ import "cosmos/base/v1beta1/coin.proto";
 option go_package = "github.com/comdex-official/comdex/x/cdp/types";
 
 // Query defines the gRPC querier service.
-service Query {
+service QueryService {
```
Comment (prashantkrv2020) [inline proto/comdex/cdp/v1alpha1/query.proto:15]: We don't need to write Service here. When the go generated code will add Server on top of it. Don't need to make it QueryServiceServer

File: proto/comdex/cdp/v1alpha1/msg.proto [hunk 3]
```diff
@@ -8,19 +8,19 @@ import "gogoproto/gogo.proto";
...
 
-  rpc CreateCDP(MsgCreateCDPRequest) returns (MsgCreateCDPResponse);
+  rpc MsgCreateCDP(MsgCreateCDPRequest) returns (MsgCreateCDPResponse);
```
Comment (prashantkrv2020) [inline proto/comdex/cdp/v1alpha1/msg.proto:13]: Like I said in the call, there is no need for appending msg on this RPC method. First MsgCreateCdp is the name for an object, not a method. Second, this method will be placed in MsgServer, so we already know these are for processing messages. This is creating unnecessary stuttering.

**Review by bsrinivas8687 — APPROVED** [review]

File: proto/comdex/cdp/v1alpha1/msg.proto (hunk 1 above)
Comment (dheerajkd30) [reply proto/comdex/cdp/v1alpha1/msg.proto:11]: It's fine, have seen this convention been followed. Let's follow it.

**Pull request merged**